src/main/java/uk/gov/hmcts/darts/arm/api/ArmDataManagementApi.java

```diff
@@ -1,8 +1,19 @@
 package uk.gov.hmcts.darts.arm.api;
 
 import com.azure.core.util.BinaryData;
+import com.azure.storage.blob.models.BlobItem;
+
+import java.util.Map;
 
 public interface ArmDataManagementApi {
 
     String saveBlobDataToArm(String filename, BinaryData binaryData);
+
+    Map<String, BlobItem> listCollectedBlobs(String prefix);
```

Comment:
Not really an issue seeing as we are heavily bound to Azure but would be good to keep our internal apis cloud agnostic